src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -2745,6 +2747,74 @@ bool ViewProviderSketch::doubleClicked(void)
     return true;
 }
 
+QString ViewProviderSketch::getPresentationString(const Constraint *constraint)
+{
+    Base::Reference<ParameterGrp>   hGrpSketcher; // param group that includes HideUnits option
+    bool                            iHideUnits;
+    QString                         userStr; // final return string
+    QString                         unitStr;  // the actual unit string
+    QString                         baseUnitStr; // the expected base unit string
+    double                          factor; // unit scaling factor, currently not used
+    Base::UnitSystem                unitSys; // current unit system
+    
+    // Get value of HideUnits option. Default is false.
+    hGrpSketcher = App::GetApplication().GetUserParameter().GetGroup("BaseApp")->GetGroup("Preferences")->GetGroup("Mod/Sketcher");
+    iHideUnits = hGrpSketcher->GetBool("HideUnits", 0l);

```

Comment:
This sets the default to true, although in the ui the default is false (unchecked) Should be the same everywhere. I'd recommend the default to be false, so existing users don't experience an unwanted change

Reply:
Good catch. Typo on my part: comments state what I meant to do as well as the GUI.

I'll update it when I get home unless it's an easy thing for you to fix (not sure what it looks like from your side).